packages/nunjucks/README.md

```diff
@@ -20,7 +20,7 @@ npm i @frctl/nunjucks --save
 
 ```javascript
 fractal.components.engine("@frctl/nunjucks"); // register the Nunjucks adapter for your components
-fractal.components.set("ext", ".nunj"); // look for files with a .nunj file extension
+fractal.components.set("ext", ".njk"); // look for files with a .nunj file extension
```

Comment:
https://mozilla.github.io/nunjucks/templating.html#file-extensions

njk seems to be the community-adopted default so I think we should recommend it as well?